main/docs/fr-ca/get-started/apis/delete-api-permissions.mdx

```diff
@@ -16,7 +16,7 @@ Vous pouvez supprimer les autorisations d’une API à l’aide d’Auth0 Dash
 
 ## Utiliser le Dashboard
 
-1. Rendez-vous dans [Dashboard > Applications > AP](/docs/images//manage.auth0.com/#/apis) et cliquez sur le nom de l’API pour l’afficher.
+1. Rendez-vous dans [Auth0 Dashboard > Applications > APIs](https://manage.auth0.com/#/apis) et cliquez sur le nom de l’API pour l’afficher.
```

Comment:
🤔 This is the first time we've corrected from "Dashboard" to "Auth0 Dashboard", which is the correct verbiage?